esp/src/eclwatch/templates/WUDetailsWidget.html

```diff
@@ -36,6 +36,7 @@
                                         <option value="High">${i18n.High}</option>
                                     </select>
                                     <input id="${id}AllowForeignFiles" title="${i18n.AllowForeignFiles}:" name="AllowForeignFiles" colspan="2" data-dojo-props="trim: true" checked data-dojo-type="dijit.form.CheckBox" />
+                                    <input id="${id}UpdateSuperFiles" title="${i18n.UpdateSuperFiles}:" name="UpdateSuperFiles" colspan="2" data-dojo-props="trim: true" data-dojo-type="dijit.form.CheckBox" />
```

Comment:
You can remove trim from checkbox. Also, do you want a default state on checkbox? If not, this code is fine. If you wish you can add `checked="true"`